pkg/elastic/elastic_search_api_service.go

```diff
@@ -41,7 +41,20 @@ func NewElasticSearchService(settings *config.Settings, logger zerolog.Logger) (
 	var netClient = &http.Client{
 		Timeout: time.Second * 10,
 	}
-	return &elasticSearchService{
+	return &ElasticSearchService{
```

Comment:
Hmm, I think we must be careful here. "App search" is the separate thing that powers the model autocomplete in the app. You want the "analytics host", I think. Might be best as a separate client? They don't have much to do with each other.

Reply:
Correct, you'll need to add different settings too. 
[Example here](https://github.com/DIMO-Network/device-data-api/blob/e3db1632083d7df73c0fb2cf319c1397876e1b63/internal/config/settings.go#L11-L14), and you can trace back the settings for the client - might be a good one to put into the [shared](https://github.com/DIMO-Network/shared) repo.